lib/eventFactory.js

```diff
@@ -145,17 +154,23 @@ function startBuild(config) {
 function createBuilds(config) {
     const { eventConfig, eventId, pipeline, pipelineConfig, changedFiles, webhooks } = config;
     const startFrom = eventConfig.startFrom;
+    let branch = '';
 
     if (!startFrom) {
         return null;
     }
 
-    return pipeline.jobs.then((jobs) => {
+    return pipeline.branch.then((b) => {
```

Comment:
Can you call `pipeline.branch` and `pipeline.jobs` at the same time using `Promise.all` like:
```
return Promise.all([
                pipeline.branch,
                pipeline.jobs
            ]).then(([branch, jobs]) => {
```

Reply:
I will fix. Thank you!!